conf/messages.en

```diff
@@ -834,6 +834,11 @@ trustee.individual.passportDetailsYesNo.checkYourAnswersLabel = Do you know {0}
 trustee.individual.passportDetails.checkYourAnswersLabel = What are {0}’s passport details?
 trustee.individual.idCardDetailsYesNo.checkYourAnswersLabel = Do you know {0}’s ID card details?
 trustee.individual.idCardDetails.checkYourAnswersLabel = What are {0}’s ID card details?
+trustee.individual.5mld.countryOfNationalityInTheUkYesNo.checkYourAnswersLabel = Does {0} have UK nationality?
+trustee.individual.5mld.countryOfNationality.checkYourAnswersLabel = What is {0}’s country of nationality?
+trustee.individual.5mld.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel = Is {0} a UK resident?
+trustee.individual.5mld.countryOfResidence.checkYourAnswersLabel = What is {0}’s country of residence?
+trustee.individual.5mld.legallyIncapableYesNo.checkYourAnswersLabel = Does {0} have mental capacity at the time of registration?
```

Comment:
Will hold off on merging until you decide whether you're going to rename the legally incapable stuff to mental capacity